src/main/java/gregtech/api/items/gui/PlayerInventoryHolder.java

```diff
@@ -56,6 +56,10 @@ public boolean isRemote() {
         return player.getEntityWorld().isRemote;
     }
 
+    public ItemStack getSampleItem() {
+        return sampleItem;
+    }
+
```

Comment:
use getCurrentItem() plz.

Reply:
I think I can do that, although that does mean that I will have to remove the Precondition in the constructor, as MetaItems aren't able to use that.